tests/eval/ja/test_reference_free_text_quality.py

```diff
@@ -68,13 +69,20 @@ def test_toxicity_openai(generated_outputs):
 
 
 @pytest.mark.parametrize('generated_outputs',
-                         [['ご機嫌いかがですか？私はとても元気です。', '機嫌いかが？私はとても元気人です。'], ['猫']])
+                         [
+                             ['ご機嫌いかがですか？私はとても元気です。',
+                              '機嫌いかが？私はとても元気人です。'
+                             ], ['猫'], '猫'
+                         ])
 def test_fluency(generated_outputs):
     eval_value = fluency(generated_outputs)
     assert all(0 <= v <= 1 for v in eval_value.metric_values)
 
 
-@pytest.mark.parametrize('generated_outputs', ['ご機嫌いかがですか？私はとても元気です。'])
+@pytest.mark.parametrize('generated_outputs',
+                         ['ご機嫌いかがですか？私はとても元気です。',
+                          ['ご機嫌いかがですか？私はとても元気です。']
```

Comment:
I think this is what you intended?

```suggestion
                          'ご機嫌いかがですか？私はとても元気です。'
```

Reply:
I actually wanted to test that the method works for both single string & array so what I wrote is right I think

Reply:
Ah I misread the brackets in the Github diff. You're right!